docs/html/user_guide.rst

```diff
@@ -772,6 +772,172 @@ archives are built with identical packages.
++------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``*``      | Can be used at the end of a version number to represent "all"                                                                                                         | **``== 3.1.*``:** any version that starts with ``3.1``. Equivalent to ``~=3.1.0``.                                                                            |
++------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+The detailed specification of supported comparison operators can be
+found in `PEP440 <https://www.python.org/dev/peps/pep-0440/>`__.
+
+Possible solutions
+------------------
+
+The solution to your error will depend on your individual use case. Here
+are some things to try:
+
+1. Audit your top level requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As a first step it is useful to audit your project and remove any
+unneccessary or out of date requirements. Removing these can
+significantly reduce the complexity of your dependency tree, thereby
+reducing opportunities for conflicts to occur.
+
+2. Loosen your top level requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes the packages that you have asked pip to install are
+incompatible because you have been too strict when you specified the
+package version.
+
+In our first example both ``packageA`` and ``packageB`` have been
+*pinned* to use specific versions
+(``packageA==0.44.1b0 packageB==4.3.0``).
+
+To find a version of both ``packageA`` and ``packageB`` that depend on
+the same version of ``packageC``, you might consider:
+
+-  Loosening the range of packages that you are prepared to install
+   (e.g. ``pip install "packageA>0.44.*" "packageB>4.0.0"``)
+-  Asking pip to install *any* version of ``packageA`` and ``packageB``
+   by removing the version specifiers altogether (e.g.
+   ``pip install packageA packageB``)
+
+In the second case, pip will automatically find a version of both
+``packageA`` and ``packageB`` that depend on the same version of
+``packageC``, installing:
+
+-  ``packageA 0.46.0b0``, which depends on ``packageC 2.6.1``
+-  ``packageB 4.3.0`` which *also* depends on ``packageC 2.6.1``
+
+If you want to prioritise one package over another, you can add version
```

Comment:
I am fine with the British spelling here and appreciate it. :-)

Reply:
huhu, didn't even notice it @brainwane!

In the [Warehouse docs](https://warehouse.readthedocs.io/ui-principles/#write-clearly-with-consistent-style-and-terminology) we say "For consistency, we prefer using American English spellings over British English."

I'm pretty sure we'll use the Warehouse docs as a starting point for a general packaging copywriting styleguide, so I am actually going to change this in anticipation of that.